projects/plugins/jetpack/class.jetpack-cli.php

```diff
@@ -230,12 +232,12 @@ public function disconnect( $args, $assoc_args ) {
 				);
 				break;
 			case 'user':
-				if ( ( new Connection_Manager( 'jetpack' ) )->disconnect_user( $user->ID ) ) {
+				if ( ( new Connection_Manager( 'jetpack' ) )->disconnect_user( $user->ID, $force_user_disconnect ) ) {
 					Jetpack::log( 'unlink', $user->ID );
 					WP_CLI::success( __( 'User has been successfully disconnected.', 'jetpack' ) );
 				} else {
 					/* translators: %s is a username */
-					WP_CLI::error( sprintf( __( "User %s could not be disconnected. Are you sure they're connected currently?", 'jetpack' ), "{$user->login} <{$user->email}>" ) );
+					WP_CLI::error( sprintf( __( "User %s could not be disconnected. Are you sure they're currently connected?\nNOTE: If you're trying to disconnect the connection owner, you need to use --force", 'jetpack' ), "{$user->data->user_login} <{$user->data->user_email}>" ) );
```

Comment:
Is there a better way to error? Could we detect if it is the connection owner and state that instead?

Reply:
Sure there is. I did this quickly because I was focused on some last minute user-less bugs. I'll mark this as in progress and wrap it up properly.

Reply:
Done and updated the test instructions